code/datums/status_effects/neutral.dm

```diff
@@ -40,11 +40,34 @@
 
 /datum/status_effect/high_five
 	id = "high_five"
-	duration = 40
+	duration = 5 SECONDS
 	alert_type = null
 
-/datum/status_effect/high_five/on_remove()
-	owner.visible_message("[owner] was left hanging....")
+/datum/status_effect/high_five/proc/get_missed_message()
+	var/list/missed_highfive_messages = list(
+		"it looks like [owner.p_they()] [owner.p_were()] left hanging...",
+		"seeming to wave at nobody in particular.",
+		"moving [owner.p_their()] hand directly to [owner.p_their()] forehead in shame.",
+		"fully committing and high-fiving empty space.",
+		"high-fiving [owner.p_their()] other hand shamefully before wiping away a tear.",
+		"going for a handshake, then a fistbump, before pulling [owner.p_their()] hand back...? What [owner.p_are()] [owner.p_they()] doing?"
+	)
+
+	return pick(missed_highfive_messages)
+
+/datum/status_effect/high_five/before_remove(success = FALSE)
+	. = TRUE  // return true so we don't cancel the removal
+	if(success)
+		// we should have already showed our success message
+		return .
+
+	// show some emotionally damaging failure messages
+	// high risk, high reward
+	owner.visible_message("[owner] awkwardly lowers [owner.p_their()] hand, [get_missed_message()]")
+
+/datum/status_effect/high_five/on_timeout()
+	. = ..()
+	before_remove(FALSE)
```

Comment:
why put this functionality in `before_remove` why not just have it in `on_timeout()` ?
and you don't need the parent call on `on_timeout()` 
just have the ontimeout do the 
`owner.visible_message("[owner] awkwardly lowers [owner.p_their()] hand, [get_missed_message()]")`

Reply:
Well when the high-five succeeds we need to deliberately remove the highfive trait without posting this message, and when *highfive gets called again we deliberately remove it to allow people to cancel it (which tbh might be just worth removing, and would simplify this).